packages/atomic/src/pages/index.html

```diff
@@ -167,7 +167,7 @@
             <atomic-numeric-facet field="sncost" label="Cost Range (auto)">
               <atomic-format-currency currency="CAD"></atomic-format-currency>
             </atomic-numeric-facet>
-            <atomic-timeframe-facet label="Timeframe" with-date-picker>
+            <atomic-timeframe-facet sort-criteria="descending" label="Timeframe" with-date-picker>
```

Comment:
Since default is `descending`, this is not strictly necessary.

Reply:
```suggestion
            <atomic-timeframe-facet label="Timeframe" with-date-picker>
```